### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: #3293

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: src/tabs/adjustments.html
Comment (McGiverGim) [inline src/tabs/adjustments.html:5]: Where go this buttons with the new code?

Comment (haslinghuis) [reply src/tabs/adjustments.html:5]: Attributes are managed dynamically in gui.js

https://github.com/betaflight/betaflight-configurator/blob/master/src/js/gui.js#L303-L311

**Review by McGiverGim — APPROVED** [review]

File: locales/en/messages.json
Comment (nerdCopter) [inline locales/en/messages.json:3215]: non-breaking but maybe a space before `target`.
image

**Review by nerdCopter — APPROVED** [review]

File: locales/en/messages.json
Comment (haslinghuis) [reply locales/en/messages.json:3215]: Thanks. Updated.

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**
